src/sentry/static/sentry/app/views/releasesV2/detail/overview/issues.tsx

```diff
@@ -25,22 +27,17 @@ class Issues extends React.Component<Props, State> {
     issuesType: 'new',
   };
 
-  // TODO(releasesV2): figure out the query we want + do we want to pass globalSelectionHeader values?
   getDiscoverUrl() {
-    const {version, orgId} = this.props;
+    const {version, orgId, projectId} = this.props;
 
     const discoverQuery = {
       id: undefined,
       version: 2,
       name: `${t('Release')} ${formatVersion(version)}`,
-      fields: ['title', 'count(id)', 'event.type', 'user', 'last_seen'],
-      query: `release:${version}`,
-
-      projects: [],
-      range: '',
-      start: '',
-      end: '',
-      environment: [''],
+      fields: ['title', 'count(id)', 'event.type', 'issue', 'last_seen'],
```

Comment:
@matejminar  We've recently updated the `count()` and `last_seen()` column fields. Their old forms may no longer work in the future.

```suggestion
      fields: ['title', 'count()', 'event.type', 'issue', 'last_seen()'],
```

Reply:
Thanks!